save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/utils/FileUtil.kt

```diff
@@ -0,0 +1,21 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package org.cqfn.save.preprocessor.utils
+
+import java.io.File
+import java.math.BigInteger
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.security.DigestInputStream
+import java.security.MessageDigest
+
+private const val RADIX = 16
+
+/**
+ * @return hash of file content
+ */
+fun File.toHash(): String {
+    val md = MessageDigest.getInstance("MD5")
+    Files.newInputStream(Paths.get(this.path)).use { inputSrteam -> DigestInputStream(inputSrteam, md) }
+    return BigInteger(1, md.digest()).toString(RADIX)
```

Comment:
Pls add a test, where hash is calculated

Reply:
It wasn't covered, because passed wrong path to directory, now it is covered by test

Reply:
Orchestrator should receive relative path; please change walking method to be called with absolute, if necessary

Reply:
Fixed